lib/cf/dashboard/src/test/authmiddleware-test.js

```diff
@@ -125,6 +125,14 @@ describe('lib', () => {
         sinon.assert.calledWith(passportAuthStub,req,res,next);
         done();
       });
+
+      it('should not authenticate /healthcheck call', (done) => {
```

Comment:
The rest of Abacus healthchecks use basic strategy for authentication. Can we adopt the same here?

Reply:
Yes, we can adopt the same strategy for dashboard also.